content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,214 @@
+   this time. (Since no services have been performed, nothing has been *delivered*, therefore there
+   there is nothing to invoice).
+
+Create a helpdesk ticket for time-tracked services
+--------------------------------------------------
+
+To record a :guilabel:`Timesheet` entry, go to the :menuselection:`Helpdesk` app and select the
+appropriate :guilabel:`Team`.
+
+If there is already an existing ticket for this issue, select it from the kanban view to go to the
+details form. If there is no existing ticket for this customer issue, click :guilabel:`New` to
+create a new ticket and enter the necessary customer information.
+
+In the :guilabel:`Sales Order Line` drop-down, select the :abbr:`SO (sales order)` created in the
+previous step.
+
+Track support hours on a ticket
+-------------------------------
+
+In order to create an invoice for a product based on timesheets, hours need to be tracked and
+recorded. At this point, the service is considered *delivered*. To record hours for this support
+service, click on the :guilabel:`Timesheets` tab of the :guilabel:`Helpdesk ticket`.
+
+Click :guilabel:`Add a Line` to record a new entry. Select an :guilabel:`Employee` from the
+drop-down, and record the :guilabel:`Hours Spent`.
+
+Repeat these steps as needed until all time spent on the issues has been recorded.
+
+.. image:: track_and_bill/track-bill-record-timesheet-hours.png
+   :align: center
+   :alt: View of the timesheets tab on a helpdesk ticket.
+
+Create an invoice for hours tracked on a ticket
+-----------------------------------------------
+
+After the customer's issue has been solved, and it is determined no additional timesheet entries
+will be made, an invoice can be created and the customer can be billed.
+
+To do this, return to the :abbr:`SO (sales order)` by clicking on the :guilabel:`Sales Order` smart
+button at the top of the :guilabel:`Helpdesk Ticket`.
+
+Before creating the invoice, confirm that the number in the :guilabel:`Delivered` column matches
+the total number of :guilabel:`Hours Spent` listed in the :guilabel:`Timesheets` tab on the ticket.
+
+.. image:: track_and_bill/track-bill-delivered-timesheet-hours.png
+   :align: center
+   :alt: View of a sales order with emphasis on the delivered column.
+
+Then click :guilabel:`Create Invoice`. This will open a :guilabel:`Create Invoices` pop-up window.
+If a :guilabel:`Down Payment` needs to be collected, change the :guilabel:`Create Invoice` type.
+
+.. important::
+   Use the :guilabel:`Timesheets Period` field if this invoice should only include timesheets from
+   a certain time period. If this field is left blank, *all* applicable timesheets that have not
+   yet been invoiced will be included.
+
+.. image:: track_and_bill/track-bill-create-invoice-timesheets-period.png
+   :align: center
+   :alt: View of create invoices pop up showing timesheets period fields.
+
+Click :guilabel:`Create Invoice`. The invoice can now be sent to the customer for payment.
```

Comment:
When the necessary information has been entered into the appropriate fields, click :guilabel:`Create Draft Invoice`.

Doing so means this invoice can now be sent to the customer for payment.